app/auth/gitlab_auth.py

```diff
@@ -23,12 +23,8 @@
 from urllib.parse import urljoin
 
 import jwt
-from oic import rndstr
-from oic.oauth2.grant import Token
-from oic.oic import Client
-from oic.oic.message import AuthorizationResponse, RegistrationResponse
-from oic.utils.authn.client import CLIENT_AUTHN_METHOD
-from oic.utils.keyio import KeyJar
+
+from requests_oauthlib import OAuth2Session
```

Comment:
Can we cleanup the unused imports here? The following imports are not used:
* json
* urllib
* jwt
* Oauth2Session
* Response
* session
* get_redis_key_from_session
* RenkuWebApplicationClient